src/SprykerEco/Client/Payone/PayoneFactory.php

```diff
@@ -26,7 +32,7 @@ class PayoneFactory extends AbstractFactory
      *
      * @return \SprykerEco\Client\Payone\ClientApi\Call\CreditCardCheck
      */
-    public function createCreditCardCheckCall(array $defaults)
+    public function createCreditCardCheckCall(array $defaults): CreditCardCheckInterface
```

Comment:
* Fix the return type hint in the document block.

Reply:
++